openmdao/docs/usr-guide/tutorials/implicit.rst

```diff
@@ -0,0 +1,251 @@
+.. _`solver_tutorial`:
+
+Solving an Implicit Relationship with a Newton Solver
+=====================================================
+
+This tutorial will show how to use a solver to drive some unknown in your
+model to zero by varying a param. In OpenMDAO 0.x, you did this by adding a
+driver (e.g., Newton), putting the relevant components into its workflow, and
+adding parameters for the driven params and equality constraints for the
+expression to be driven to zero. In OpenMDAO 1.x, you do this by adding a
+component with an implicit equation.
+
+We will illustrate this with a simple problem.
+
+Find the Intersection of a Parabola and a Line
+----------------------------------------------
+
+Consider a parabola defined by "y = 3x^2 - 5" and a line defined by "y = -2x + 4".
+We would like to find their intersection.
+
+.. figure:: intersect.png
+   :align: center
+   :alt: Intersection of a line and parabola (courtesy of Wolfram Alpha).
+
+Plugging this into Wolfram Alpha gives the solution above. We will implement
```

Comment:
don't point to wolfram alpha... regenerate image in matplotlib with slightly better quality.